rivet.sh

```diff
@@ -6,12 +6,14 @@ requires:
   - HepMC3
   - YODA
   - fastjet
+  - cgal
```

Comment:
For the record, `requires` implies `build_requires`.

Reply:
apparently without cgal in the build_requires GCC is not able to find the CGAL library during building

Reply:
Edit: I had made a typo, it works as expected locally